ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/security/trustedlaunch/provision/cucushift-installer-rehearse-azure-ipi-security-trustedlaunch-provision-chain.yaml

```diff
@@ -0,0 +1,17 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-security-trustedlaunch-provision
+  steps:
+  - chain: ipi-conf-azure
+  - ref: ipi-conf-azure-confidential
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-azure-confidential
+  env:
+    - name: SECURITY_TYPE
+      default: "TrustedLaunch"
+      documentation: |-
+        Specifies the SecurityType of the virtual machine
+        Valid Values: "ConfidentialVM","TrustedLaunch".
```

Comment:
So we do not have plan to cover `ConfidentialVM` yet, right?

Reply:
will add it after blocked bugs is fixed.